### DISCUSSION TITLE
ASReview installation problem (errorno2)

### DISCUSSION THREAD

**Discussion by bosselaar** [asreview/asreview#589] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
My ASReview installation failed, due to a so-called 'environment error', producing this message: 
![image](https://user-images.githubusercontent.com/82939461/115536153-c2fd2c80-a299-11eb-8e5f-867f2c15d308.png)

It appeared to be a path length problem 
So I reinstalled Python and disabled 'path length limit' during the installation process and the installation of ASReview was successful.

**Comment by Rensvandeschoot** [comment, marked as answer]
Dear @bosselaar, thank you for letting us know!
